misc/userscripts/qute-pass

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
```

Comment:
Can you please also add a warning similar to [what password_fill has](https://github.com/qutebrowser/qutebrowser/blob/be325853d81ba17568ef3caa30dc840b884fc7fd/misc/userscripts/password_fill#L10-L13) that mentions that passwords land in qutebrowser's debug log? Either as a comment, or as [epilog](https://docs.python.org/3/library/argparse.html#epilog) or so for ArgumentParser would be fine, I think.

Reply:
Also, please add a copyright/GPL header ([example](https://github.com/qutebrowser/qutebrowser/blob/be325853d81ba17568ef3caa30dc840b884fc7fd/qutebrowser/app.py#L3-L18)) - you can use your name/email there.

Reply:
Done and done.